components/DateInput/index.tsx

```diff
@@ -1,30 +1,29 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import dayjs from 'dayjs';
 import COLORS from '@/styles/colors';
 import { Flex } from '@/styles/containers';
 import { P2 } from '@/styles/text';
-import { DropdownIcon, SelectContainer } from '../CustomSelect/styles';
 import Icon from '../Icon';
-import { DateInputWrapper, HiddenDateInput } from './styles';
+import { DropdownIcon } from './styles';
 
 interface DateInputProps {
   label?: string;
   value: string;
   onChange: (value: string) => void;
   placeholder?: string;
-  min?: string;
-  max?: string;
 }
 
 export default function DateInput({
   label,
   value,
   onChange,
   placeholder = '',
-  min,
-  max,
 }: DateInputProps) {
   const containerRef = useRef<HTMLDivElement | null>(null);
-  const hiddenInputRef = useRef<HTMLInputElement | null>(null);
+  const [isDatePickerOpen, setIsDatePickerOpen] = useState(false);
 
```

Comment:
Its not letting me leave a comment on the useEffect on line 29 but like is the event listener doing anything? It looks like an empty if statement to me?